website/docs/r/http_source.html.markdown

```diff
@@ -43,7 +51,7 @@ resource "sumologic_collector" "collector" {
 In addition to the common properties, the following arguments are supported:
 
 - `message_per_request` - (Optional) When set to `true`, will create one log message per HTTP request.
-- `content_type`        - (Optional) When configuring a HTTP Traces Source, set this property to `Zipkin`. This should only be used when creating a Traces source.
+- `content_type`        - (Optional) When configuring a HTTP Traces Source, set this property to `Zipkin` and `KinesisLog` for Kinesis Log source. This should only be used when creating a Traces or Kinesis Log source.
```

Comment:
The phrasing is a bit confusing. Can we change to 
> When configuring an HTTP Traces Source, set this property to `Zipkin`. When configuring a Kinesis Log source, set this property to  `KinesisLog`.

Reply:
Done